src/Tickets/Emails/Email/Purchase_Receipt.php

```diff
@@ -245,8 +245,8 @@ public function send() {
 		}
 
 		$placeholders = [
-			'{order_number}' => $order->ID,
-			'{order_id}'     => $order->ID,
+			'{order_number}' => $order->gateway_order_id,
+			'{order_id}'     => $order->gateway_order_id,
```

Comment:
I suggest we use the `$order->ID` instead of `gateway_order_id` as the gateway order ID are random and doesn't look good.

Reply:
As you can see this screenshot is showing the PayPal order number which to me looks bad: https://share.cleanshot.com/nfDhLc9w

Also, if we send the order number on our end this will help the admin to figure out the Order from order list easily: https://share.cleanshot.com/sY8CNwz8